static/app/components/core/badge/alertBadge.chonk.tsx

```diff
@@ -0,0 +1,62 @@
+import {css, type DO_NOT_USE_ChonkTheme, type SerializedStyles} from '@emotion/react';
+
+import type {AlertBadgeProps} from 'sentry/components/core/badge/alertBadge';
+import {chonkStyled} from 'sentry/utils/theme/theme.chonk';
+import {IncidentStatus} from 'sentry/views/alerts/types';
+
+function makeChonkAlertBadgeDiamondBackgroundTheme(
+  status: AlertBadgeProps['status'],
+  isIssue: AlertBadgeProps['isIssue'],
+  isDisabled: AlertBadgeProps['isDisabled'],
+  theme: DO_NOT_USE_ChonkTheme
+): SerializedStyles {
+  if (isDisabled) {
+    return css`
+      color: ${theme.textColor};
+      background: ${theme.colors.dynamic.surface500};
+      border: 1px solid ${theme.colors.dynamic.surface100};
+    `;
+  }
```

Comment:
I think we talked about this in the other PR as well that returning `css` as a `SerializedStyles` isn’t type-safe, and we concluded that we’d rather want to return an object that is of type `CssProperties` ?

Reply:
Ah yes, I need to change this indeed. The reason it's here is that I was using a descendant SVG selector, but I managed to remove that

Reply:
FYI, I made a custom type on top of `CssProperties` that will be able to handle those, too:

https://github.com/getsentry/sentry/pull/85943/files#diff-f0219135c18e4530eedc13c70b9de34db3d574a7a141afdd7e2de89990983915R1130-R1134